mne/io/eeglab/eeglab.py

```diff
@@ -285,14 +292,14 @@ class RawEEGLAB(_BaseRaw):
     @verbose
     def __init__(self, input_fname, montage, eog=(), event_id=None,
                  event_id_func='strip_to_integer', preload=False,
-                 verbose=None):
+                 verbose=None, uint16_codec=None):
         """Read EEGLAB .set file.
         """
         from scipy import io
         basedir = op.dirname(input_fname)
         _check_mat_struct(input_fname)
         eeg = io.loadmat(input_fname, struct_as_record=False,
-                         squeeze_me=True)['EEG']
+                         squeeze_me=True, uint16_codec=uint16_codec)['EEG']
```

Comment:
what version of `scipy` do you use? I don't have this argument? we might have to do some version checking for this ...